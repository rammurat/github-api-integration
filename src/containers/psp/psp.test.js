import React from 'react'
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'

import PSP from './psp'

const store = {
    subscribe: () => { },
    dispatch: () => { },
    getState: () => ({
        posts: {
            initialConfig: {
                pagination: {
                    perPage: 0
                }
            }
        }
    })
}

// @ts-ignore
it('renders correctly', () => {
    const tree = renderer
        // @ts-ignore
        .create(<Provider store={store}><PSP /></Provider>)
        .toJSON();
    // @ts-ignore
    expect(tree).toMatchSnapshot();
});