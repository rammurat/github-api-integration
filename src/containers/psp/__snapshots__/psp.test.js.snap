// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bwzfXH iwjKrt"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-12 col-md-3 col-lg-3"
      >
        <div>
          <form>
            <h1>
              Filters:
            </h1>
            <div
              className="form-group"
            >
              <label
                htmlFor="language-type"
              >
                Language Type:
              </label>
              <select
                className="form-control"
                id="language-type"
                name="language-type"
                onChange={[Function]}
              />
            </div>
            <div
              className="form-group"
            >
              <label
                htmlFor="sort-by"
              >
                Sort by stars:
              </label>
              <select
                className="form-control"
                id="sort-by"
                name="sort-by"
                onChange={[Function]}
              />
            </div>
          </form>
        </div>
      </div>
      <div
        className="col-sm-12 col-md-9 col-lg-9"
      >
        <form
          className="form-inline search-container"
          onSubmit={[Function]}
        >
          <div
            className="form-group mx-sm-3 mb-2"
          >
            <label
              className="sr-only"
            >
              Password
            </label>
            <input
              className="form-control"
              id="search"
              name="search"
              onChange={[Function]}
              placeholder="JavaScript"
              type="text"
              value=""
            />
          </div>
          <button
            className="btn btn-primary mb-2"
            type="submit"
          >
            Search
          </button>
        </form>
        <div>
          <div
            className="row justify-content-end"
          >
            <div
              className="col-sm-12 col-md-6 col-lg-6"
            />
            <div
              className="col-sm-12 col-md-6 col-lg-6"
            >
              <nav
                aria-label="Page navigation example"
                className="product-pagination "
              >
                <ul
                  className="pagination"
                >
                  <li
                    className="page-item"
                  >
                    <a
                      className="page-link"
                      href="#"
                    >
                      Previous
                    </a>
                  </li>
                  <li
                    className="page-item"
                  >
                    <a
                      className="page-link"
                      href="#"
                    >
                      1
                    </a>
                  </li>
                  <li
                    className="page-item"
                  >
                    <a
                      className="page-link"
                      href="#"
                    >
                      2
                    </a>
                  </li>
                  <li
                    className="page-item"
                  >
                    <a
                      className="page-link"
                      href="#"
                    >
                      3
                    </a>
                  </li>
                  <li
                    className="page-item"
                  >
                    <a
                      className="page-link"
                      href="#"
                    >
                      Next
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
          <div
            className="spinner-border"
            role="status"
          >
            <span
              className="sr-only"
            >
              Loading...
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
